.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
+        }
+      }
+    ],
+    "@babel/preset-typescript",
+    "@babel/preset-react"
+  ],
+  "plugins": []
+}
```

Comment:
Would it be possible to rename this file to `.babelrc.storybook.json` to avoid confusion? and specifically call it from the `stroybook`s scripts?

The docs seems to be outdated for the CLI since they keep calling it `start-storybook` https://storybook.js.org/docs/7.0/react/api/cli-options

Reply:
A babel config file is now needed for the parsing to work in storybook, as there is no longer a configuration out of the box and uses the project's configuration instead.

This config file is the default generated with the migration process run in the CLI. Without it and the added deps, an error throws pointing to `./node_modules/@storybook/builder-webpack5/node_modules/babel-loader/lib/index.js`

Here are the [details from the migration docs](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#babel-mode-v7-exclusively)

I have not seen anything in the Gatsby discord or in their GH issues regarding any suggested alternatives to this generated config.

Reply:
That's ok. Yeah, I was asking just to add clarity. At least wouldn't be clear to me that this new `babelrc` file is only used for storybook and not for Gastby for example.

Reply:
Right. Renaming the file would be the way to go here.

Reply:
Changed the way of loading it.